workloads/network-perf/common.sh

```diff
@@ -3,9 +3,8 @@
 source env.sh
 source ../../utils/common.sh
 source ../../utils/benchmark-operator.sh
-source ../../utils/compare.sh
 
-openshift_login
+openshift_login 
```

Comment:
extra space at end of line can be removed.

Reply:
should be removed now